### PR TITLE
Add test for CollectionEditVerbManager

### PR DESCRIPTION
**Pull request by LeafShi1**
Related #10773

## Proposed changes

- Add test for CollectionEditVerbManager.cs

## Customer Impact

- No

## Regression? 

-  No

## Risk

- Minimal

## Test methodology 

- Unit tests

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12088)

### CODE REVIEW

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#nullable enable
+
+using System.ComponentModel;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class CollectionEditVerbManagerTests
+{
+    public static TheoryData<string?, string> ValuesTheoryData()
+        => new()
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs:13]:
```suggestion
    public static TheoryData<string?, string> ValuesTheoryData() => new()
```

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        {
+            { null, SR.ToolStripItemCollectionEditorVerb },
+            { "TestText", "TestText" }
+        };
+
+    [Theory]
+    [MemberData(nameof(ValuesTheoryData))]
+    public void CollectionEditVerbManager_EditItemsVerb_ReturnExpectedValue(string? text, string expectedValue)
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs:21]:
```suggestion
    public void CollectionEditVerbManager_EditItemsVerb_ReturnExpectedValue(string? text, string expected)
```

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        using ToolStripButton toolStripButton = new();
+        ToolStripItemDesigner toolStripItemDesigner = new();
+        toolStripItemDesigner.Initialize(toolStripButton);
+
+        PropertyDescriptorCollection properties = TypeDescriptor.GetProperties(typeof(ToolStripButton));
+        PropertyDescriptor? propertyDescriptor = properties["Text"];
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs:28]:
```suggestion
        PropertyDescriptor? propertyDescriptor = properties[nameof(ToolStripButton.Text)];
```

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+        CollectionEditVerbManager editor = new(text, toolStripItemDesigner, propertyDescriptor, true);
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs:30]: Please add argument name before the literal.

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            { null, SR.ToolStripItemCollectionEditorVerb },
+            { "TestText", "TestText" }
+        };
+
+    [Theory]
+    [MemberData(nameof(ValuesTheoryData))]
+    public void CollectionEditVerbManager_EditItemsVerb_ReturnExpectedValue(string? text, string expectedValue)
+    {
+        using ToolStripButton toolStripButton = new();
+        ToolStripItemDesigner toolStripItemDesigner = new();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs:24]:
```suggestion
        using ToolStripItemDesigner toolStripItemDesigner = new();
```

File: src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+        PropertyDescriptorCollection properties = TypeDescriptor.GetProperties(typeof(ToolStripButton));
+        PropertyDescriptor? propertyDescriptor = properties["Text"];
+
+        CollectionEditVerbManager editor = new(text, toolStripItemDesigner, propertyDescriptor, true);
+        editor.EditItemsVerb.Text.Should().Be(expectedValue);
+        editor.EditItemsVerb.Enabled.Should().BeTrue();
+        editor.EditItemsVerb.Supported.Should().BeTrue();
+        editor.EditItemsVerb.Visible.Should().BeTrue();
+        editor.EditItemsVerb.CommandID!.ID.Should().Be(8192);
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs:35]: Is it possible to replace a hardcoded number with TestAccessor reading a property?

Comment (LeafShi1) [reply src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/CollectionEditVerbManagerTests.cs]: > Is it possible to replace a hardcoded number with TestAccessor reading a property?

The CommandID is System.ComponentModel.Design.CommandID, there is no Command definition in the Winform repo

**Review by Tanya-Solyanik — APPROVED** [review]

**Pull request merged**